### PR TITLE
[Xedra Evolved] Moon's tears weapons

### PR DESCRIPTION
**Pull request by Standing-Storm**
#### Summary
Mods "[Xedra Evolved] Moon's tears weapons"

#### Purpose of change

As I add increasingly more character types who have trouble with iron or steel, there should be weapons that they can use. Also, surely not every changeling bannerman or pooka aspirant has an enchanted blade. Some of them just have weapons that are lighter than steel and impossibly sharp.

Also, the moon's tears ingot was autolearn, which is a bit weird when no other ingot is.

#### Describe the solution

Add a variety of moon's tears weapons, recipes for them, and have various Fair Folk drop them. Currently, only trow changelings can learn to forge moon's tears weapons because I couldn't think of another suitable place to put the recipes.

Add a `Report: LAUGHING DANCER metallurgy` item to Border Patrol offices that has the moon's tears ingot recipe. 

#### Testing

Weapons exist, have appropriate stats, and drop from changelings.

#### Additional context

I keep thinking of adding Volund/Weyland Smith as an NPC who can forge all kinds of impossible items for you, but that would be a lot of work.

### CODE REVIEW

File: data/mods/Xedra_Evolved/items/melee_moons_tears.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  },
+  {
+    "type": "TOOL",
+    "id": "moons_tears_mace",
+    "copy-from": "ch_mace",
+    "looks_like": "ch_mace",
+    "name": { "str_sp": "moon's tears mace" },
+    "description": "A medieval weapon consisting of a wood handle with a heavy metal head forged from some silvery metal that seems to shine with a soft internal light.  It looks delicate but hits very hard.",
+    "replace_materials": { "ch_steel": "moon_tears" },
+    "proportional": { "melee_damage": { "bash": 1.2 }, "weight": 0.8 }
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/items/melee_moons_tears.json:41]: If this mace isn't enchanted, it being lighter than steel yet hitting harder doesn't make sense 🤔

File: data/mods/Xedra_Evolved/recipes/blacksmithing.json [hunk 2]
```diff
@@ -75,9 +87,201 @@
...
+  },
+  {
+    "type": "recipe",
+    "activity_level": "BRISK_EXERCISE",
+    "result": "moons_tears_broadsword",
+    "category": "CC_*",
+    "subcategory": "CSC_*_NESTED",
+    "skill_used": "fabrication",
+    "difficulty": 8,
+    "time": "19 h",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/recipes/blacksmithing.json:131]:
```suggestion
    "time": "19 h",
    "flags": [ "SECRET" ],
```

File: data/mods/Xedra_Evolved/recipes/blacksmithing.json [hunk 3]
```diff
@@ -75,9 +87,201 @@
...
+  },
+  {
+    "type": "recipe",
+    "activity_level": "BRISK_EXERCISE",
+    "result": "moons_tears_arming_sword",
+    "category": "CC_*",
+    "subcategory": "CSC_*_NESTED",
+    "skill_used": "fabrication",
+    "difficulty": 8,
+    "time": "22 h",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/recipes/blacksmithing.json:151]:
```suggestion
    "time": "22 h",
    "flags": [ "SECRET" ],
```

File: data/mods/Xedra_Evolved/recipes/blacksmithing.json [hunk 4]
```diff
@@ -75,9 +87,201 @@
...
+  },
+  {
+    "type": "recipe",
+    "activity_level": "BRISK_EXERCISE",
+    "result": "moons_tears_mace",
+    "category": "CC_*",
+    "subcategory": "CSC_*_NESTED",
+    "skill_used": "fabrication",
+    "difficulty": 5,
+    "time": "15 h",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/recipes/blacksmithing.json:172]:
```suggestion
    "time": "15 h",
    "flags": [ "SECRET" ],
```

File: data/mods/Xedra_Evolved/recipes/blacksmithing.json [hunk 5]
```diff
@@ -75,9 +87,201 @@
...
+  },
+  {
+    "type": "recipe",
+    "activity_level": "BRISK_EXERCISE",
+    "result": "moons_tears_scimitar",
+    "category": "CC_*",
+    "subcategory": "CSC_*_NESTED",
+    "skill_used": "fabrication",
+    "difficulty": 9,
+    "time": "19 h",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/recipes/blacksmithing.json:193]:
```suggestion
    "time": "19 h",
    "flags": [ "SECRET" ],
```

File: data/mods/Xedra_Evolved/recipes/blacksmithing.json [hunk 6]
```diff
@@ -75,9 +87,201 @@
...
+  },
+  {
+    "type": "recipe",
+    "activity_level": "BRISK_EXERCISE",
+    "result": "moons_tears_estoc",
+    "category": "CC_*",
+    "subcategory": "CSC_*_NESTED",
+    "skill_used": "fabrication",
+    "difficulty": 9,
+    "time": "20 h 10 m",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/recipes/blacksmithing.json:213]:
```suggestion
    "time": "20 h 10 m",
    "flags": [ "SECRET" ],
```

File: data/mods/Xedra_Evolved/recipes/blacksmithing.json [hunk 7]
```diff
@@ -75,9 +87,201 @@
...
+  },
+  {
+    "result": "moons_tears_spear",
+    "type": "recipe",
+    "activity_level": "BRISK_EXERCISE",
+    "category": "CC_*",
+    "subcategory": "CSC_*_NESTED",
+    "skill_used": "fabrication",
+    "difficulty": 5,
+    "time": "8 h",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/recipes/blacksmithing.json:233]:
```suggestion
    "time": "8 h",
    "flags": [ "SECRET" ],
```

File: data/mods/Xedra_Evolved/recipes/blacksmithing.json [hunk 8]
```diff
@@ -75,9 +87,201 @@
...
+  },
+  {
+    "type": "recipe",
+    "activity_level": "BRISK_EXERCISE",
+    "result": "moons_tears_battleaxe",
+    "category": "CC_*",
+    "subcategory": "CSC_*_NESTED",
+    "skill_used": "fabrication",
+    "difficulty": 8,
+    "time": "14 h",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/recipes/blacksmithing.json:254]:
```suggestion
    "time": "14 h",
    "flags": [ "SECRET" ],
```

File: data/mods/Xedra_Evolved/recipes/blacksmithing.json [hunk 9]
```diff
@@ -75,9 +87,201 @@
...
+  },
+  {
+    "type": "recipe",
+    "activity_level": "BRISK_EXERCISE",
+    "result": "moons_tears_katana",
+    "category": "CC_*",
+    "subcategory": "CSC_*_NESTED",
+    "skill_used": "fabrication",
+    "difficulty": 9,
+    "time": "20 h",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/recipes/blacksmithing.json:275]:
```suggestion
    "time": "20 h",
    "flags": [ "SECRET" ],
```

**Review by TheShadowFerret — CHANGES_REQUESTED** [review]
The mace being lighter yet hitting harder than hardened steel doesn't make sense if it's not enchanted🤔

Otherwise just making sure all of them got the SECRET flag rather than just the baselard

**Comment by Standing-Storm** [comment]
> The mace being lighter yet hitting harder than hardened steel doesn't make sense if it's not enchanted🤔

As the report says

> Some of these described specifications don't seem physically possible

Thank you for catching the secret flags!

**Comment by Naadn** [comment]
Feel like there should be more way to obtain moon tear scrap beside murder tons of changing just for them drop moon tear chainmail and uncraft them all.

**Comment by Maleclypse** [comment]
One of the things I want to add are mineral outcroppings that appear in mines so there are a few mine variants that have moon tears. Some controlled by zombies, some by changelings and others experiencing fatal weirdness.

**Comment by Standing-Storm** [comment]
> Feel like there should be more way to obtain moon tear scrap beside murder tons of changing just for them drop moon tear chainmail and uncraft them all.

There should, but to avoid scope creep (and since this PR is already almost 500 lines), I just added the weapons. The same reason I didn't add any more armor.

**Pull request merged**